spec/models/step_spec.rb

```diff
@@ -124,4 +124,41 @@
       expect(step_item.broken_links.length).to eql 1
     end
   end
+
+  describe '.request_broken_links' do
+    context 'when there are links in the step' do
+      it 'should have an associated LinkReport with a batch id' do
+        link_checker_api_create_batch(
+          uris: [
+            'https://www.gov.uk/good/stuff',
+            "https://www.gov.uk/also/good/stuff",
+            "https://www.gov.uk/not/as/great",
+            "http://example.com/good"
+          ],
+          id: 1234,
+          webhook_uri: "https://collections-publisher.test.gov.uk/link_report"
```

Comment:
The webhook isn't added until the next commit, so perhaps this line should be added there?